### PR TITLE
build: footer + fix jumbotron theme

### LINKED ISSUE #72: [ENHANCEMENT]: Implement footer
### Description

The footer does not exist, we must implement it.

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by LeoFVO**
Closes #72  

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code is documented
- [ ] I provided unitary tests or procedure to test my code
- [ ] My PR have a clear description of what my code is supposed to do

### CODE REVIEW

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 1]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+              underline="none"
+              variant="body1"
+              sx={{ display: 'block' }}
+              color="inherit"
+            >
+              Cours
+            </Link>
+          </Grid>
+          <Grid xs={6} lg={3}>
+            <Typography variant="h5">Mon compte</Typography>
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:50]: i18n

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 2]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+            <Link
+              href="/users/profile/videos"
+              underline="none"
+              variant="body1"
+              sx={{ display: 'block' }}
+              color="inherit"
+            >
+              Mon profil
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:58]: i18n

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 3]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+            </Link>
+            <Link
+              href="/users/profile/settings"
+              underline="none"
+              variant="body1"
+              color="inherit"
+            >
+              Paramètre
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:66]: i18n

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 4]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+            <Typography variant="h5">Assistance</Typography>
+            <Link
+              href="https://github.com/polyflix/polyflix/blob/main/CGU-Polyflix.pdf"
+              underline="none"
+              variant="body1"
+              sx={{ display: 'block' }}
+              color="inherit"
+              target="_blank"
+            >
+              Conditions générale d&apos;utilisation
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:79]: i18n

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 5]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+            </Link>
+            <Link
+              href="#"
+              underline="none"
+              variant="body1"
+              sx={{ display: 'block' }}
+              color="inherit"
+              target="_blank"
+            >
+              Mentions légales
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:89]: i18n

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 6]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+            </Link>
+            <Link
+              href="https://docs.google.com/forms/d/e/1FAIpQLScy8UUG38btVXtym4UTBWrJKaOAMRA8-zY2yxCCeUyYolTjOA/viewform"
+              underline="none"
+              variant="body1"
+              sx={{ display: 'block' }}
+              color="inherit"
+              target="_blank"
+            >
+              Signaler un bug
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:99]: i18n

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 7]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+          </Grid>
+          <Grid
+            xs={6}
+            lg={3}
+            display="flex"
+            justifyContent="center"
+            alignItems="center"
+          >
+            <Typography variant="body1">
+              &copy; Copyright Polyflix 2022 Tout droits réservés.
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:110]: i18n

**Review by MaloPolese — CHANGES_REQUESTED** [review]

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 8]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+              Navigation
+            </Typography>
+            <Link
+              href="/"
+              underline="none"
+              variant="body1"
+              sx={{ display: 'block' }}
+              color="inherit"
+            >
+              Page d&apos;accueil
```
Comment (hugoamalric) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:28]: i18n

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 9]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
     >
-      {/* TODO */}
       <PageSection>
-        <Typography variant="body1" color="initial">
-          Footer
-        </Typography>
+        <Grid container spacing={2}>
+          <Grid xs={6} lg={3}>
+            <Typography variant="h5" component={'p'}>
+              Navigation
```
Comment (hugoamalric) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:19]: i18n

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 10]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+              Page d&apos;accueil
+            </Link>
+            <Link
+              href="/videos/explore"
+              underline="none"
+              variant="body1"
+              sx={{ display: 'block' }}
+              color="inherit"
+            >
+              Vidéos
```
Comment (hugoamalric) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:37]: i18n

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 11]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+            </Link>
+            <Link
+              href="/courses/explore"
+              underline="none"
+              variant="body1"
+              sx={{ display: 'block' }}
+              color="inherit"
+            >
+              Cours
```
Comment (hugoamalric) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:46]: i18n

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 12]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+              underline="none"
+              variant="body1"
+              sx={{ display: 'block' }}
+              color="inherit"
+              target="_blank"
+            >
+              Mentions légales
+            </Link>
+            <Link
+              href="https://docs.google.com/forms/d/e/1FAIpQLScy8UUG38btVXtym4UTBWrJKaOAMRA8-zY2yxCCeUyYolTjOA/viewform"
```
Comment (hugoamalric) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:92]: maybe put this link in a constant if we want to change it some day

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 13]
```diff
@@ -12,11 +12,105 @@ export const Footer = () => {
...
+              variant="body1"
+              color="inherit"
+            >
+              Paramètre
+            </Link>
+          </Grid>
+          <Grid xs={6} lg={3}>
+            <Typography variant="h5">Assistance</Typography>
+            <Link
+              href="https://github.com/polyflix/polyflix/blob/main/CGU-Polyflix.pdf"
```
Comment (hugoamalric) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:72]: maybe put this link in a constant if we want to change it some day

**Review by hugoamalric — CHANGES_REQUESTED** [review]

**Comment by MaloPolese** [comment]
image

Do a footer like this

**Comment by MaloPolese** [comment]
image

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 14]
```diff
@@ -12,10 +15,109 @@ export const Footer = () => {
         mt: 4,
       }}
     >
-      {/* TODO */}
       <PageSection>
-        <Typography variant="body1" color="initial">
-          Footer
+        <Grid container spacing={2}>
+          <Grid xs={6} md={4}>
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:20]: This grid is useless

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 15]
```diff
@@ -12,10 +15,109 @@ export const Footer = () => {
         mt: 4,
       }}
     >
-      {/* TODO */}
       <PageSection>
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:18]: remove grid below and use flex on section

File: apps/polyflix/public/locales/en/common.json [hunk 16]
```diff
@@ -49,6 +49,25 @@
...
+      "courses": "Courses"
+    },
+    "account": {
+      "title": "My account",
+      "profile": "My profile",
+      "settings": "My settings"
+    },
+    "help": {
+      "title": "Assistance",
+      "cgu": "Terms of use",
```
Comment (hugoamalric) [inline apps/polyflix/public/locales/en/common.json:66]:
```suggestion
      "cgu": "Terms of service",
```

File: apps/polyflix/public/locales/en/common.json [hunk 17]
```diff
@@ -49,6 +49,25 @@
...
+      "videos": "Videos",
+      "courses": "Courses"
+    },
+    "account": {
+      "title": "My account",
+      "profile": "My profile",
+      "settings": "My settings"
+    },
+    "help": {
+      "title": "Assistance",
```
Comment (hugoamalric) [inline apps/polyflix/public/locales/en/common.json:65]:
```suggestion
      "title": "Support",
```
sounds better

File: apps/polyflix/public/locales/en/common.json [hunk 18]
```diff
@@ -49,6 +49,25 @@
...
+      "cgu": "Terms of use",
+      "mentions": "Legal mentions",
```
Comment (hugoamalric) [inline apps/polyflix/public/locales/en/common.json:67]:
```suggestion
      "mentions": "Terms and conditions",
```
or
```suggestion
      "mentions": "Legal notice",
```

File: apps/polyflix/public/locales/fr/common.json [hunk 19]
```diff
@@ -49,6 +49,25 @@
...
+      "courses": "Cours"
+    },
+    "account": {
+      "title": "Mon compte",
+      "profile": "Mon profil",
+      "settings": "Mes paramètres"
+    },
+    "help": {
+      "title": "Assistance",
+      "cgu": "Conditions générale d'utilisation",
```
Comment (hugoamalric) [inline apps/polyflix/public/locales/fr/common.json:66]:
```suggestion
      "cgu": "Conditions générales d'utilisation",
```

**Review by hugoamalric — CHANGES_REQUESTED** [review]

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 20]
```diff
@@ -12,10 +15,109 @@ export const Footer = () => {
...
+              color="inherit"
+            >
+              {t('footer.navigation.courses')}
+            </Link>
+          </Grid>
+          <Grid xs={6} md={4}>
+            <Typography variant="h6" paddingBottom={1}>
+              {t('footer.account.title')}
+            </Typography>
+            <Link
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:56]: use a list <ul></ul>

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 21]
```diff
@@ -1,7 +1,67 @@
...
+        {
+          title: t('footer.help.cgu'),
+          href: 'https://github.com/polyflix/polyflix/blob/main/CGU-Polyflix.pdf',
+        },
+        {
+          title: t('footer.help.mentions'),
+          href: '#',
+        },
+        {
+          title: t('footer.help.bug'),
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:51]: you can juste put the string path and use t() inside the template

**Review by MaloPolese — CHANGES_REQUESTED** [review]

**Comment by MaloPolese** [comment]
> Fix lint

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 22]
```diff
@@ -12,10 +64,41 @@ export const Footer = () => {
...
-          Footer
+        <Stack
+          direction={{ xs: 'column', sm: 'row' }}
+          spacing={2}
+          justifyContent="space-evenly"
+          paddingBottom={2}
+        >
+          {links.map((section, index) => (
+            <List key={index}>
+              <Typography variant="h6" component={'p'} paddingBottom={1}>
```
[the remark this answers is not in the document]
Comment (MaloPolese) [reply apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:84]: with css

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 23]
```diff
@@ -12,10 +72,42 @@ export const Footer = () => {
         mt: 4,
       }}
     >
-      {/* TODO */}
       <PageSection>
-        <Typography variant="body1" color="initial">
-          Footer
+        <Stack
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:76]: items layout
vertial: center
horizontal: space around

**Review by MaloPolese — CHANGES_REQUESTED** [review]

**Review by MaloPolese — APPROVED** [review]
[❤️](https://emojipedia.org/red-heart/)
image

**Review by hugoamalric — APPROVED** [review]

**Pull request merged**